data/json/items/armor.json

```diff
@@ -10933,7 +10933,7 @@
         "name" : "briefcase",
         "weight" : 1700,
         "color" : "light_gray",
-        "covers" : ["ARM_EITHER", "HAND_EITHER"],
+        "covers" : ["ARM_EITHER", "HAND_EITHER", "RESTRICT_HANDS"],
```

Comment:
You probably need to add "RESTRICT_HANDS" to flags, not covered body parts. Same for bindle, plastic shopping bag and straw basket.